packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -546,4 +546,207 @@
         <protected>false</protected>
         <shortDescription>:</shortDescription>
     </labels>
+    <labels>
```

Comment:
@jpmarceau Please review this file. Most of the strings were copied from atomic, so they should already be correct.